seaplane-sdk/python/src/seaplane/smartpipes/build.py

```diff
@@ -2,35 +2,37 @@
 import os
 from typing import Any, Dict
 
+from ..logging import log
 from .decorators import context
-from .executor import SchemaExecutor
+from .executor import RealCoProcessorExecutor, SchemaExecutor
```

Comment:
```suggestion
from .executor import RealCoprocessorExecutor, SchemaExecutor
```
plus renaming the function itself